features/user/reset_password.feature

```diff
@@ -8,6 +8,7 @@ Scenario: Request password reset
   And I click 'Send reset email' button
   Then I see a success banner message containing 'send a link to reset the password'
   And I receive a 'reset-password' email for that user.emailAddress
+  And I wait 2 seconds
```

Comment:
It might be nice to add an optional inline comment to this step so that you can give some explanation about why, i.e. `And I wait 2 seconds to ensure the reset token is valid`